smibhid/main.py

```diff
@@ -4,14 +4,23 @@
 """
 # Built against Pico W Micropython firmware v1.22.2 https://micropython.org/download/RPI_PICO_W/
 
-from lib.config.config_management import ConfigManagement
-config_management = ConfigManagement()
-config_errors = config_management.check_config()
+from os import listdir
 
-from lib.hid import HID
+updates_dir_list = listdir("/updates/")
```

Comment:
Doesn't run when the updates directory doesn't exist

Reply:
Put a try except in to handle missing folder for simplicity of main.py